src/congpairs.cc

```diff
@@ -31,6 +31,7 @@ using libsemigroups::Congruence;
 using libsemigroups::word_t;
 using libsemigroups::relation_t;
 using libsemigroups::RecVec;
+using libsemigroups::Partition;
```

Comment:
Fine, just wanted to make sure it wasn't some extraneous code that you'd forgotten to delete.